.cloud-build/tests/select_notebooks_test.py

```diff
@@ -0,0 +1,33 @@
+import sys
+sys.path.append("..")
+
+from execute_changed_notebooks_helper import (load_results, select_notebook)
+
+bucket:str = "cloud-build-notebooks-presubmit"
+bucket_file: str = "build_results"
+accum = {}
```

Comment:
Unit tests are supposed to be independent, so don't include global mutable state.

Reply:
done